rbac/management/role/definer.py

```diff
@@ -32,14 +32,15 @@
 def _make_role(tenant, data):
     """Create the role object in the database."""
     name = data.pop("name")
+    display_name = data.pop("display_name", name)
     access_list = data.get("access")
     defaults = dict(
         description=data.get("description", None),
         system=True,
         version=data.get("version", 1),
         platform_default=data.get("platform_default", False),
     )
-    role, created = Role.objects.get_or_create(name=name, defaults=defaults)
+    role, created = Role.objects.get_or_create(name=name, display_name=display_name, defaults=defaults)
```

Comment:
Do we want the `display_name` as part of the `get_or_create`? Won't this make it so if the `display_name` changes in the config it will create a new role vs update it? Then it will fail because of the name constraint (currently fails silently).

I think we may need to leave `display_name` out of the `get_or_create`, and have it as part of the `update` on line 48?
```
Role.objects.filter(name=name).update(**defaults, display_name=display_name)
```

Reply:
I think you're right that we'll need to add it to the update. In testing I ran into some errors with it not in get_or_create, but it's been a few days so I'll have to go back and see what exactly the behavior was. Will hop on that first thing tomorrow.